decidim-accountability/app/forms/decidim/accountability/admin/result_form.rb

```diff
@@ -40,7 +40,10 @@ def map_model(model)
         end
 
         def proposals
-          @proposals ||= Decidim.find_resource_manifest(:proposals).try(:resource_scope, current_feature)&.order(title: :asc)&.pluck(:title, :id)
+          @proposals ||= Decidim.find_resource_manifest(:proposals)
+                                .try(:resource_scope, current_feature)
```

Comment:
Fix identation 😄

Reply:
I think it's a bug that rubocop didn't catch this. I reported it: https://github.com/bbatsov/rubocop/issues/5621...